Makefile

```diff
@@ -11,7 +11,13 @@ clean:
 
 test: unit e2e
 
-generate:
+MOCK_FILES=pkg/util/mocks/mock_upgrade/types.go pkg/util/mocks/mock_arm/arm.go
+
+pkg/util/mocks/mock_%.go: pkg/%.go
```

Comment:
any chance this can be done via `//go:generate` ?

Reply:
and can you update hack/validate-generated.sh (it would be nice if it could be changed such that it didn't need to be updated in a case like this)

Reply:
will do, I have a look at go generate